cohortextractor/cohortextractor.py

```diff
@@ -365,6 +365,12 @@ def main():
         choices=["all"] + [x[0] for x in list_study_definitions()],
         default="all",
     )
+    generate_cohort_parser.add_argument(
+        "--temp-database-name",
+        help="Name of database so store temporary results",
```

Comment:
Yeah, env var is better but I didn't know how end users were going to be setting env vars and I know they must be able to pass args so I figured this would at least allow me to explain what to do